apis/nucleus/src/components/listbox/components/ListBoxRowColumn.jsx

```diff
@@ -136,6 +136,7 @@ const Root = styled('div')(({ theme }) => ({
       display: 'flex',
       alignItems: 'center',
       paddingLeft: 0,
+      minWidth: '70%', // might need proper calculation
```

Comment:
Why was this needed?

Reply:
to make able the checkbox label span element to occupy the necessary space in order to move the "A" according to justifyContent=textAlign - otherwise it will be always placed on the left 

I will check if I can slightly change it with minWidth and some padding on the right